ci/infra/libvirt/cloud-init/common.tpl

```diff
@@ -42,7 +42,7 @@ bootcmd:
   - ip link set dev eth0 mtu 1400
   # Hostnames from DHCP - otherwise localhost will be used
   - /usr/bin/sed -ie "s#DHCLIENT_SET_HOSTNAME=\"no\"#DHCLIENT_SET_HOSTNAME=\"yes\"#" /etc/sysconfig/network/dhcp
-  - netconfig update -f
+  # - netconfig update -f
```

Comment:
This is a test to address the zypper update failure, right? I think it shouldn't be part of this PR.